django-data/image/validation/tests/test_tasks.py

```diff
@@ -112,8 +149,10 @@ def check_async_not_called(self):
         self.assertEqual(self.send_msg_ws.call_count, 0)
 
     def tearDown(self):
+        # stopping mock objects
         self.asyncio_mock_patcher.stop()
         self.send_msg_ws_patcher.stop()
+        self.create_vsummary_patcher.stop()
```

Comment:
`tearDown` is a method called after each test. `start` and `stop` patcher to have `call_count` consistent in very test (`!= 0` if mock is called during test)

Reply:
👍